lua/neogit/lib/finder.lua

```diff
@@ -9,6 +9,13 @@ local function refocus_status_buffer()
   end
 end
 
+local copy_selection = function()
+  local selection = require("telescope.actions.state").get_selected_entry()
+  if selection ~= nil then
+    vim.cmd.let(("@+='%s'"):format(selection[1]))
```

Comment:
I think you could use `%q` instead of `'%s'` to let :format() handle quoting and escaping here.